modules/openapi-generator/src/main/java/org/openapitools/codegen/CodegenModel.java

```diff
@@ -64,7 +64,7 @@
     public String defaultValue;
     public String arrayModelType;
     public boolean isAlias; // Is this effectively an alias of another simple type
-    public boolean isString, isInteger, isLong, isNumber, isNumeric, isFloat, isDouble, isDate, isDateTime, isShort, isUnboundedInteger;
+    public boolean isString, isInteger, isLong, isNumber, isNumeric, isFloat, isDouble, isDate, isDateTime, isShort, isUnboundedInteger, isBoolean;
```

Comment:
The following function also needs to be updated as well:

https://github.com/OpenAPITools/openapi-generator/blob/5d58afe5e79c425452716d48dc7f6c2bf39147d7/modules/openapi-generator/src/main/java/org/openapitools/codegen/CodegenModel.java#L764

https://github.com/OpenAPITools/openapi-generator/blob/5d58afe5e79c425452716d48dc7f6c2bf39147d7/modules/openapi-generator/src/main/java/org/openapitools/codegen/CodegenModel.java#L854

https://github.com/OpenAPITools/openapi-generator/blob/5d58afe5e79c425452716d48dc7f6c2bf39147d7/modules/openapi-generator/src/main/java/org/openapitools/codegen/CodegenModel.java#L872

I'll update these in another PR after merging this one.

Reply:
Filed https://github.com/OpenAPITools/openapi-generator/pull/9802